### PR TITLE
Andrewrutter/kkb 48 waiver page frontend

### PR DESCRIPTION
**Pull request by andlrutt**
This one is super simple. Made a waiver page based off of the existing one on [BetterUnite](https://www.betterunite.com/Document/0651420e-cb3b-4cff-9d27-d9c544437d9e) with a few small changes, and added the link to it on the EventSignUp component. I do however have one thing that I'd appreciate if you looked over:

The bottom text of the waiver on BetterUnite ("BY ACKNOWLEDGING THIS WAIVER I CERTIFY...") cuts off at the end, so I grabbed some stuff that seemed like it belonged there off the internet. I'd appreciate you reading it as a sanity check, I don't want to mess around with legalese all that much. Everything else has been copy-pasted so it should be fine.

image

### CODE REVIEW

File: src/components/EventSignUp/index.tsx [hunk 1]
```diff
@@ -111,7 +111,9 @@ export default function EventSignUp() {
                             <br></br>
                             <DescriptionIcon htmlColor="gray" />
                             &nbsp;
-                            <Link className={styles.waiverLink}>{constants.org.name.short} Volunteer Waiver</Link>
+                            <Link href="/waiver" target="_blank" className={styles.waiverLink}>
```
Comment (kemalfidan) [inline src/components/EventSignUp/index.tsx]: Nice, I like how it'll open in a new tab.

**Review by kemalfidan — APPROVED** [review]
Looks good. Can you also put a comment in `utils/constants` under `org`? It just needs to say that the waiver would need to be changed for different orgs.

**Review by zaviermiller — APPROVED** [review]
100

**Review by rossketron — APPROVED** [review]
Looks good!

**Pull request merged**